aptos-move/aptos-gas/src/gas_meter.rs

```diff
@@ -257,6 +266,32 @@ impl AptosGasMeter {
         }
     }
 
+    #[inline]
+    fn charge_execution(&mut self, amount: InternalGas) -> PartialVMResult<()> {
+        self.charge(amount)?;
+
+        self.execution_gas_used += amount;
+        if self.feature_version >= 7
+            && self.execution_gas_used > self.gas_params.txn.max_execution_gas
```

Comment:
I don't see max_execution_gas or max_io_gas added to the gas schedule. How are they configured?